api.md

```diff
@@ -779,7 +779,237 @@ Get all chunks older than 3 months and newer than 4 months:
 SELECT show_chunks(older_than => interval '3 months', newer_than => interval '4 months');
 ```
 
+## reorder_chunk() [](reorder_chunk)
+Reorder a chunk's heap to follow the order of an index. This function acts
+similarly to the PostgreSQL CLUSTER command (see:
+https://www.postgresql.org/docs/current/sql-cluster.html), however it uses lower
+lock levels so that the chunk and hypertable are able to be read for most of the
+process. It does use a bit more disk space during the operation. This command
+can be particularly useful for data that is accessed when it is older in a
+significantly different order than it was originally inserted in. In general
+data in the heap in a hypertable is organized in loose time order, much as it
+was inserted. But, for instance, if one has a `device_id` column and usually
+accesses data by `device_id`, an index on `device_id, time` would be a good
+candidate for reordering. 
+
+One can call this function directly, but using `add_reorder_policy` is often much more convenient
+
+#### Required Arguments [](reorder_chunk-required-arguments)
+
+|Name|Description|
+|---|---|
+| `chunk` | (REGCLASS) Name of the chunk to reorder. |
+
+#### Optional Arguments [](reorder_chunk-optional-arguments)
+
+|Name|Description|
+|---|---|
+| `index` | (REGCLASS) The name of the index (on either the hypertable or chunk) to order by.|
+| `verbose` | (BOOLEAN) Setting to true will display messages about the progress of the reorder command. Defaults to false.|
+
+#### Returns [](reorder_chunk-returns)
+
+This function returns void. 
+
+
+#### Sample Usage [](reorder_chunk-examples)
+
+
+```sql
+SELECT reorder_chunk('_timescaledb_internal._hyper_1_10_chunk', 'conditions_device_id_time_idx');
+```
+
+runs a reorder on the `_timescaledb_internal._hyper_1_10_chunk` chunk using the `conditions_device_id_time_idx` index. 
+
+---
+
+## Policies
+The following functions allow the administrator to create/remove/alter policies
+that schedule administrative actions to take place on a hypertable. The actions
+are meant to implement data retention or perform tasks that will improve query
+performance on older chunks. The actions are taken and scheduled by background
+worker processes managed by Timescale. Each policy is assigned a scheduled job
+which will be run in the background to enforce it. 
+
+
+
+## add_drop_chunks_policy() [](add_drop_chunks_policy)
+Create a policy to drop chunks older than a given interval of a particular hypertable on a schedule in the background. (See drop_chunks). This implements a data retention policy and will remove data on a schedule. Only one drop_chunks policy may exists per hypertable. 
+
+#### Required Arguments [](add_drop_chunks_policy-required-arguments)
+
+|Name|Description|
+|---|---|
+| `hypertable` | (REGCLASS) Name of the hypertable to create the policy for. |
+| `older_than` | (INTERVAL) Chunks fully older than this interval when the policy is run will be dropped|
```

Comment:
"Chunks with all(only?) data older than..."?

Reply:
not exactly true, unfortunately, we're not looking at the data, we're looking at the constraints. We use this language elsewhere, so I just used it here.